package.json

```diff
@@ -0,0 +1,21 @@
+
+{
+  "name": "react-native-vitals",
+  "version": "0.0.1",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [
+    "react-native"
+  ],
+  "author": "Ying Hang Eng",
+  "license": "MIT",
+  "devDependencies": {
+    "react-native": "^0.27"
```

Comment:
There's a mismatch here between the dependencies and package version with the build.gradle file.

Let's keep them synced up!

Reply:
I think the idea is to require a generic react native dependency but I'm not sure if you can do that in `package.json`?

Reply:
@christophermark where are you seeing the version mismatch, can you link to the corresponding gradle line? I only see `compile 'com.facebook.react:react-native:+'`